core/frontend/src/views/SystemInformationView.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <v-container
+    style="max-width:90%"
+  >
+    <v-row
+      no-gutters
+    >
+      <v-col
+        md="3"
+      >
+        <v-card
+          elevation="2"
+          class="ma-3"
+        >
+          <v-navigation-drawer
+            floating
+            permanent
+          >
+            <v-list
+              dense
+              rounded
+            >
+              <v-list-item
+                v-for="item in items"
+                :key="item.title"
+                link
+                :input-value="item.value == page_selected"
+                @click="page_selected=item.value"
+              >
+                <v-list-item-icon>
+                  <v-icon>{{ item.icon }}</v-icon>
+                </v-list-item-icon>
+
+                <v-list-item-content>
+                  <v-list-item-title>{{ item.title }}</v-list-item-title>
+                </v-list-item-content>
+              </v-list-item>
+            </v-list>
+          </v-navigation-drawer>
+        </v-card>
+      </v-col>
+
+      <v-col
+        v-if="true"
+        md="9"
+      >
+        <processes v-if="page_selected == 'process'" />
+        <system-condition v-if="page_selected == 'system_condition'" />
+        <network v-if="page_selected == 'network'" />
+        <about-this-system v-if="page_selected == 'about'" />
+      </v-col>
```

Comment:
Do we want to consider browser history (being able to go back and forth) on those submenus? If so, we should implement routes here. If not, it is ok as is.

Reply:
I think that is not necessary.